0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md

```diff
@@ -101,13 +103,17 @@ The request must specify either `source_amount` or `destination_amount` but not
   "method": "quote_response",
   "id": "721e4126-98a1-4974-b35a-8a8f4655f934",
   "data": {
-    "source_connector_account": "mark",
-    "source_ledger": "example.eur-ledger.",
-    "source_amount": "10025",
-    "source_expiry_duration": "6000",
-    "destination_ledger": "example.usd-ledger.",
-    "destination_amount": "10571",
-    "destination_expiry_duration": "5000"
+    "source_connector_account":"example.eur-ledger.mark",
+    "source_ledger":"example.eur-ledger.",
+    "source_amount":"10025000000",
+    "source_expiry_duration":"6000",
+    "destination_ledger":"example.usd-ledger.",
+    "destination_amount":"9000000000",
+    "destination_expiry_duration":"5",
+    "liquidity_curve":[
+      ["10", "0"],
+      ["100000000000000000", "99799999999999"]
```

Comment:
The current implementation actually uses `[100000000000000000, 99799999999999]`, but I guess that's a bug? It should be `["100000000000000000", "99799999999999"]`, right?